frontend/awx/administration/applications/ApplicationForm.tsx

```diff
@@ -40,7 +40,7 @@ export interface IApplicationInput {
   description?: string;
 }
 
-export function CreateApplication() {
+export function CreateApplication(props: { onSuccessfulCreate?: (app: Application) => void }) {
```

Comment:
Any reason why this is an optional prop?

Reply:
Kept it optional for the unit tests, but just updated it with my latest push